.circleci/config.yml

```diff
@@ -3,18 +3,23 @@ jobs:
   build:
     working_directory: ~/ng
     docker:
-      - image: angular/ngcontainer:0.0.1
+      - image: angular/ngcontainer:0.0.2
     steps:
       - checkout
 
       - restore_cache:
           key: angular_bazel_example-{{ .Branch }}-{{ checksum "examples/rollup/yarn.lock" }}
       - run: bazel info release
+
       - run: bazel run @yarn//:yarn
-      - run: bazel build ...
-      - run: bazel test ...
+      - run: bazel build -- ... -examples/program/...
```

Comment:
Add a comment why the `-example/program/...`

Reply:
:+1: